recipes/jpcre2/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    # settings = "os", "compiler", "build_type", "arch"
```

Comment:
settings is required here, because you're building an executable.
Also, `tools.cross_building` needs the information in `self.settings`.
```suggestion
    settings = "os", "compiler", "build_type", "arch"
```